charts/gardener/gardenlet/templates/clusterrole-managed-istio.yaml

```diff
@@ -10,8 +10,6 @@ metadata:
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
 rules:
-{{- if eq (include "gardenlet.managed-istio-enabled" .) "true" }}
```

Comment:
The file name and the cluster role name still contain `managed-istio`. Do we want to get rid of them, e.g. renaming them to `istio` instead of `managed-istio`?